app/lib/frontend/templates/views/pkg/package_list.dart

```diff
@@ -230,7 +244,10 @@ d.Node _item({
         d.div(
           classes: ['packages-body'],
           children: [
-            d.div(classes: ['packages-description'], text: description),
+            d.div(
+              classes: ['packages-description'],
+              children: [d.span(text: description), ...topics],
```

Comment:
I'm wondering if `<span>` is better here than `<p>`... the later would put the topics into a separate line though...

Reply:
We actually do want it on the same line, so I guess we keep it like this.